.github/CHANGELOG.md

```diff
@@ -424,10 +457,10 @@ and requirements-ci.txt (unpinned). This latter would be used by the CI.
 
 * The QFT operation is moved to template
   [(#1548)](https://github.com/PennyLaneAI/pennylane/pull/1548)
```

Comment:
```suggestion
* The `QFT` operation has been moved, and is now accessible via `pennylane.templates.QFT`.
  [(#1548)](https://github.com/PennyLaneAI/pennylane/pull/1548)
```

@antalszava, having a read back of it now,

- I would remove the entries specifying 'this thing has been moved' if it results in no change to the API (e.g., `qml.Hamiltonian` still has the same public import location, even though the file has changed. They are not technically improvements, and I don't think they are useful to have here, since they are not user facing.

- I would keep entries that result in a breaking change to import location, but move them to 'breaking changes'. I.e., I believe the QFT change is breaking, right?

Reply:
Made the changes for QFT :+1: 

For the first point, I haven't found any specific examples in the file. Missing something or just something worth keeping in mind going forward?